beta/src/components/Layout/HomeContent.tsx

```diff
@@ -0,0 +1,512 @@
+                Illustration goes here
+              </div>
+            </div>
+
+            <div className="w-full rounded-3xl lg:gap-20 flex-col lg:flex-row flex items-center">
+              <div className="w-full lg:w-6/12 rounded-lg px-20 py-40 my-8 justify-center leading-6 overflow-x-auto flex items-center bg-wash dark:bg-gray-95 shadow-lg overflow-hidden">
+                Illustration goes here
+              </div>
+
+              <div className="flex-col p-8 w-full lg:w-6/12 text-primary dark:text-primary-dark flex grow items-end justify-start mx-auto">
+                <h3 className="w-full leading-tight font-bold text-4xl mb-6 text-left">
+                  Let a framework take care of the rest
+                </h3>
+                <p className="text-xl text-secondary dark:text-secondary-dark text-opacity-80 leading-normal text-left">
+                  React is a library. It lets you create components and put them
+                  together, but it doesn&apos;t prescribe solutions for routing
+                  and data fetching.
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <div className="bg-link bg-opacity-8 my-20 max-w-6xl mx-auto rounded-3xl p-20 text-base text-secondary dark:text-secondary-dark flex flex-col w-full justify-center items-center">
+          <Logo className="text-link dark:text-link-dark w-36 mt-12 h-auto mx-auto self-start" />
+          <div className="flex flex-wrap">
+            <div className="mb-8 mt-4">
+              <h1 className="text-center text-6xl mr-4 flex wrap font-bold leading-tight text-primary dark:text-primary-dark">
+                Welcome to the React community
+              </h1>
+              <p className="text-4xl w-full text-center text-secondary dark:text-secondary-dark leading-relaxed self-center my-2">
+                Get started today.
+              </p>
+            </div>
+          </div>
+        </div>
+
+        <div className="my-20 px-12 flex flex-col w-full">
+          <div className="p-20 text-base text-secondary dark:text-secondary-dark flex flex-col w-full justify-center items-center">
+            <Logo className="text-link dark:text-link-dark w-36 mt-12 h-auto mx-auto self-start" />
+            <div className="flex flex-wrap">
+              <div className="mb-8 mt-4">
+                <h1 className="text-center text-6xl mr-4 flex wrap font-bold leading-tight text-primary dark:text-primary-dark">
+                  Welcome to the React community
+                </h1>
+                <p className="text-4xl w-full text-center text-secondary dark:text-secondary-dark leading-relaxed self-center my-2">
+                  Get started today.
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </>
+  );
+}
+
+function Example() {
+  const [hover, setHover] = useState(false);
+  return (
+    <div
```

Comment:
Instead of using state for this? We could actually try to use group modifiers in tailwind itself?  https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state